### DISCUSSION TITLE
Após instalar o sistema qual login e senha do Operador?

### DISCUSSION THREAD

**Discussion by maickeberni** [MakersNetwork/agenda-saude#289] [General]
Qual o Login e senha do Operador?

**Comment by pedroCervi** [comment]
Localmente é:

```
login: mlabs
password: dontpanic
```

Essa info fica no arquivo `db/seeds.rb`

**Comment by maickeberni** [comment]
Obrigado.




*Maicke Henrique Couto Berni | Suporte Técnico / IT*

Berni Informatica Ltda | Av. Republica Argentina, 3169 | CEP: 85856-000 |
Foz do Iguaçu  - Pr
F: +55 45 3028 0009 <%2B55%2041%203420%201728> / 45 9-9839 5550 | E:
***@***.***

*www. <http://www.mosaicco.com.br/>compoclinic.com.br
<http://compoclinic.com.br>* |








Em sáb., 24 de abr. de 2021 às 08:25, Pedro Germano Cervi <
***@***.***> escreveu:
